app/assets/scripts/components/new-atbd/index.js

```diff
@@ -196,7 +196,11 @@ function NewAtbd() {
                 </span>
                 <div>Google Docs</div>
               </TemplateLink>
-              <TemplateLink href='/'>
+              <TemplateLink
+                href='https://github.com/NASA-IMPACT/nasa-apt/files/11421194/APT.ATBD.Template.docx'
```

Comment:
@sunu @frozenhelium I added the template file this way – this probably isn't best practice, so let me know if we (or the client) should host these templates in a different way

Reply:
@wrynearson I'm curious how exactly is this file hosted? I don't remember seeing `/files/<id>` like urls from github before.

Reply:
@sunu I was going to put the file in a comment on an issue, but when I uploaded the file, I thought I could just use the URL that GH provided in that comment. I don't know the retention policy for files hosted this way, and the file itself isn't linked in any comment (I didn't post the comment), so 🤷‍♂️

Reply:
@wrynearson May be worth putting it in google drive too just to be consistent

Reply:
Done